taskRuby.rb

```diff
@@ -0,0 +1,23 @@
+#Functions
+def ask(question)
+    print question
+    end
+    def Hrs(hr)

```

Comment:
I like how you have split these into functions that call eachother.  reduces redundancy 👍